docs/guides/summary-guides.md

```diff
@@ -15,6 +15,7 @@ The following guides provide integration instructions based on the needs and req
 | :--- | :--- |
 | [UID2 SDK for JavaScript Integration Guide](publisher-client-side.md) | This integration guide for publishers covers standard web integration scenarios that use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md). |
 | [Publisher Integration Guide, Server-Only](custom-publisher-integration.md) | This integration guide is for publishers that do not use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md). |
+| [Prebid Integration Guide](integration-prebid.md) | An integration guide for publishers who want to directly integrate with UID2 and generate identity tokens to be passed by Prebid in the RTB bid stream. |
```

Comment:
given the way i heard from Kanishk - i think Prebid is the way most pubs will use to integrate with UID2. could we move this to the first row on the table if there's no other reason not to?

Reply:
For sure. Didn't know it was the main integration method, gosh! Done.